formver.compiler-plugin/testData/diagnostics/conversion/internal.kt

```diff
@@ -0,0 +1,27 @@
+// NEVER_VALIDATE
+
+// Test function literals
+fun testFunctionLiteral() {
+    val f = <!VIPER_VERIFICATION_ERROR!>{ x: Int -> x + 1 }<!> // Anonymous function
+}
+// Test Double literals
+fun testDoubleLiteral() {
+    val x = <!VIPER_VERIFICATION_ERROR!>1.0<!>  // Double литерал
+}
+
+/*
+// Test class-related errors
+class TestClass
+fun testClassUsages() {
+    TestClass      // ResolvedQualifier
+    TestClass::class  // Class literal
+}*/
+
+// Test when expression
+fun <!VIPER_TEXT!>testWhen<!>(x: String) {
```

Comment:
I'm confused, we shouldn't give an output here, right?

Reply:
That's true, but maybe putting it a different way: what are you testing here?

Reply:
But why is this something we want to test?  We don't have an internal error here, so I think we're okay with any behaviour here